uavcan/si/sample/angular_acceleration/Scalar.1.0.uavcan

```diff
@@ -0,0 +1,3 @@
+uavcan.time.SynchronizedTimestamp.1.0 timestamp
+float32 radian_per_second_per_second
```

Comment:
Is there a reason that every `si/sample` type re-implements the corresponding `si/unit` type, instead of combining the timestamp with the corresponding `si/unit/` type?

Reply:
There is no reason and you made a good point. Replacing the native type with the corresponding unit type will enhance composability at the source level. Let me open a ticket.